main.go

```diff
@@ -478,6 +486,11 @@ func main() {
 		fail("Failed to get absolute project path, error: %s", err)
 	}
 
+	swiftPackagesPath, err := cache.SwiftPackagesPath(absProjectPath)
```

Comment:
please separate this logic to run only if xcode version is >= 11

(same in line 696 pls)